src/bundles.ts

```diff
@@ -183,15 +183,32 @@ interface CreateBundleFromFoldersOptions extends ConnectionOptions, AnalyzeFolde
  * @param source
  * @returns
  */
-async function getSupportedFiles(baseURL: string, source: string, requestId?: string): Promise<SupportedFiles> {
+async function getSupportedFiles(
+  baseURL: string,
+  source: string,
+  requestId?: string,
+  languages?: string[],
+): Promise<SupportedFiles> {
   emitter.supportedFilesLoaded(null);
   const resp = await getFilters(baseURL, source, undefined, requestId);
   if (resp.type === 'error') {
     throw resp.error;
   }
-  const supportedFiles = resp.value;
-  emitter.supportedFilesLoaded(supportedFiles);
-  return supportedFiles;
+  const supportedFilesFromApi = resp.value;
+  //Given supported languages from 'registy'
+  if (languages) {
+    let supportedFiles: SupportedFiles = {} as any;
+    supportedFiles.configFiles = supportedFilesFromApi.configFiles;
+    supportedFiles.extensions = languages;
+    //For verification only
+    supportedFiles.extensions = supportedFiles.extensions.filter(langSuffix =>
```

Comment:
Yesterday you mentioned renaming `langSuffix` to  `langExtenstion`. Maybe you can do it here too?